src/lib/containers/table/SynapseTable.tsx

```diff
@@ -168,15 +168,25 @@ export default class SynapseTable extends React.Component<
   shouldComponentUpdate(
     nextProps: QueryWrapperChildProps & SynapseTableProps,
     nextState: Readonly<SynapseTableState>,
-    nextContext: any,
   ): boolean {
-    this.disableResize()
-    return super.shouldComponentUpdate
-      ? super.shouldComponentUpdate(nextProps, nextState, nextContext)
-      : true
+    // ignore isFetching state variables when checking for change in state
```

Comment:
This code is just bonus, since I found that super.shouldComponentUpdate was always undefined (so it was always returning true).  Now I'm giving it a try.